configs/crdt/AL.yaml

```diff
@@ -23,3 +23,14 @@ links:
     page.done();
   message: clicking on tab 13 (number 15)
 
+- name: tertiary
+  url: https://alpublichealth.maps.arcgis.com/home/item.html?id=0c2185b8174646979f4abb5a45ef05c3&sublayer=3#data
+
+- name: quaternary
+  url: https://alpublichealth.maps.arcgis.com/home/item.html?id=0c2185b8174646979f4abb5a45ef05c3&sublayer=1#data
+
+- name: quinary
+  url: https://alpublichealth.maps.arcgis.com/home/item.html?id=015fd1e0d8074840ab624243a74c54c9&sublayer=1#data
+
+- name: deaths-ethnicity
```

Comment:
The website does not like dashes in file names, so it would be great to keep names standard and make this deaths_ethnicity even though these won't necessarily be going on the site

Reply:
Oh thank you, just pushed that update!